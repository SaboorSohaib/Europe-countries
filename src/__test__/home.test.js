import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { MemoryRouter as Router } from 'react-router-dom';
import HomePage from '../components/header';
import store from '../redux/configureStore';

it('It should render the homepage components correctly', () => {
  const homepage = render(
    <Router>
      <Provider store={store}>
        <HomePage />
      </Provider>
    </Router>,
  );
  expect(homepage).toMatchSnapshot();
});
