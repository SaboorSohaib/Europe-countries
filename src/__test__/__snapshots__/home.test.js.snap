// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the homepage components correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="header-container"
      >
        <a
          href="/"
        >
          <div>
            <svg
              class="back-arrow"
            >
              <use
                xlink:href="icons.svg#icon-chevron-left1"
              />
            </svg>
          </div>
        </a>
        <div />
        <div
          class="right-container"
        >
          <svg
            class="mic-setting"
          >
            <use
              xlink:href="icons.svg#icon-microphone"
            />
          </svg>
          <svg
            class="mic-setting"
          >
            <use
              xlink:href="icons.svg#icon-settings"
            />
          </svg>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="header-container"
    >
      <a
        href="/"
      >
        <div>
          <svg
            class="back-arrow"
          >
            <use
              xlink:href="icons.svg#icon-chevron-left1"
            />
          </svg>
        </div>
      </a>
      <div />
      <div
        class="right-container"
      >
        <svg
          class="mic-setting"
        >
          <use
            xlink:href="icons.svg#icon-microphone"
          />
        </svg>
        <svg
          class="mic-setting"
        >
          <use
            xlink:href="icons.svg#icon-settings"
          />
        </svg>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
